src/main/java/Database/playlistLibrary.java

```diff
@@ -77,13 +77,11 @@ private void createFile(String fp){
      */
```

Comment:
Small change, maybe add some Javadocs here/descriptions

Reply:
I can't tell which method you're highlighting here. Unless you're asking for some more comments within the methods on logic (which I definitely should add eventually lol), I don't think there's any methods missing javadocs?

Reply:
Ah sorry this is the saveFile() method, but just adding a small description would be good.